proto/cosmos/orm/v1alpha1/query.proto

```diff
@@ -0,0 +1,131 @@
+syntax = "proto3";
+
+package cosmos.orm.v1alpha1;
+
+import "google/protobuf/timestamp.proto";
+import "google/protobuf/duration.proto";
+import "google/protobuf/any.proto";
+import "cosmos/base/query/v1beta1/pagination.proto";
+
+// Query is a generic gRPC service for querying ORM data.
+service Query {
+
+  // Get queries an ORM table against an unique index.
+  rpc Get(GetRequest) returns (GetResponse);
+
+  // List queries an ORM table against an index.
+  rpc List(ListRequest) returns (ListResponse);
+}
+
+// GetRequest is the Query/Get request type.
+message GetRequest {
+  // message_name is the fully-qualified message name of the ORM table being queried.
+  string message_name = 1;
+
+  // index is the index fields expression used in orm definitions. If it
+  // is empty, the table's primary key is assumed. If it is non-empty, it must
+  // refer to an unique index.
+  string index = 2;
+
+  // values are the values of the fields corresponding to the requested index.
+  // There must be as many values provided as there are fields in the index and
+  // these values must correspond to the index field types.
+  repeated IndexValue values = 3;
+}
+
+// GetResponse is the Query/Get response type.
+message GetResponse {
+
+  // result is the result of the get query. If no value is found, the gRPC
+  // status code NOT_FOUND will be returned.
```

Comment:
what if a wrong index is used? I think we should return `InvalidRequst` in that case.

Reply:
Yes